lncore/channels.go

```diff
@@ -16,7 +15,7 @@ type LitChannelStorage interface {
 // ChannelHandle is "something" to concisely uniquely identify a channel.  Usually just a txid.
 type ChannelHandle [64]byte
 
-// ChannelState .
+// ChannelState
```

Comment:
I left the `.` in because `go vet` was yelling at me if I didn't have it because it wanted "properly formatted documentation" but since it isn't fully-defined yet it doesn't make sense to have full docs for that.

Reply:
Oops, yeah, we need a formatting guideline or something. But I can add this back in if go vet complains.